### PR TITLE
fix: Selected calendar unhandled promise when removing calendars

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

This PR fixes the unhandled promise when deleting selected calendars when removing a calendar integration.

Fixes # (issue)

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Connect a calendar
- Select calendars to check for conflicts
- Delete that integration
- The error shouldn't appear in the logs and the selected calendars should be removed from the db

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts [hunk 1]
```diff
@@ -206,7 +206,6 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
                   bookingFields: true,
                   seatsPerTimeSlot: true,
                   seatsShowAttendees: true,
-                  seatsShowAvailabilityCount: true,
```
Comment (alannnc) [inline packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts:209]: This was deleted because we didn't use it?

Comment (joeauyeung) [reply packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts:209]: Searched through the file and this wasn't being used anywhere else.

File: packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts [hunk 2]
```diff
@@ -317,18 +315,18 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
...
-          await prisma.selectedCalendar.delete({
+          await prisma.selectedCalendar.deleteMany({
             where: {
-              userId_integration_externalId: {
-                userId: user.id,
-                externalId: cal.externalId,
-                integration: cal.integration as string,
-              },
+              userId: user.id,
+              externalId: cal.externalId,
```
Comment (alannnc) [inline packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts:321]: I think keeping .delete() but don't await prisma but with a .catch() on every promise should do the tric.

File: packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts (hunk 1 above)
Comment (alannnc) [reply packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts:209]: Yeah linting can do that

File: packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts [hunk 3]
```diff
@@ -317,18 +315,18 @@ export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOp
 
       if (calendars && calendars.length > 0) {
         calendars.map(async (cal) => {
-          await prisma.selectedCalendar.delete({
+          await prisma.selectedCalendar.deleteMany({
```
Comment (alannnc) [inline packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts:318]: Something like this: 
```suggestion
         prisma.selectedCalendar.delete({
              userId_integration_externalId: {
                userId: user.id,
                externalId: cal.externalId,
                integration: cal.integration as string,
              },
            },
        }).catch(async catching in another thread);
```

**Review by alannnc — APPROVED** [review]

**Pull request merged**